docs/reference/cluster/stats.asciidoc

```diff
@@ -34,7 +34,10 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=node-filter]
 [[cluster-stats-api-query-params]]
 ==== {api-query-parms-title}
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeout]
+`timeout`::
+(Optional, <<time-units, time units>>)
+Period to wait for a response. If no response is received before the timeout
+expires, the request fails and returns an error. Defaults to no timeout.
```

Comment:
The timeout is used when reaching out to all nodes in the cluster, setting a timeout per node. If any of the nodes fail to respond within the timeout, the stats from that node is left out of the response. A failure indication is embedded into the response in the [`_nodes` field](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-stats.html#cluster-stats-api-response-body).

> the request fails and returns an error.

I think this is incorrect or at least imprecise. The request succeeds but includes information about nodes that failed to respond.